<launch>
    <node
        pkg="wetexplorer_vision"
        type="ring_detector.py"
        name="ring_detector"
        output="screen">
        <param name="model" value="path_to_ring2.tflite" />
        <param name="frame_width" value="640" />
        <param name="frame_height" value="480" />
        <param name="num_threads" value="4" />
        <param name="enable_edgetpu" value="false" />
    </node>
</launch>
